tests/framework/cli/test_cli.py

```diff
@@ -333,15 +332,8 @@ def test_project_commands_no_clipy(self, mocker, fake_metadata):
         )
         kedro_cli = KedroCLI(fake_metadata.project_path)
         print(kedro_cli.project_groups)
-        assert len(kedro_cli.project_groups) == 6
-        assert kedro_cli.project_groups == [
-            catalog_cli,
-            jupyter_cli,
-            pipeline_cli,
-            micropkg_cli,
-            project_group,
-            registry_cli,
-        ]
+        assert len(kedro_cli.project_groups) == 1
```

Comment:
It might be helpful to update the test name or leave a comment explaining that the 1 command group loaded is everything that's not doing lazy loading.